examples/js/exporters/GLTFExporter.js

```diff
@@ -57,7 +57,12 @@ THREE.GLTFExporter.prototype = {
 	 */
 	parse: function ( input, onDone, options ) {
 
-		options = options || {};
+		var DEFAULT_OPTIONS = {
+			trs: false,
```

Comment:
@mrdoob yep, it was already implemented https://github.com/mrdoob/three.js/blob/dev/examples/js/exporters/GLTFExporter.js#L706-L730